Sources/BraintreePayPalMessaging/BTPayPalMessagingView.swift

```diff
@@ -68,9 +67,14 @@ public class BTPayPalMessagingView: UIView {
                 style: PayPalMessageStyle(
                     logoType: request.logoType.logoTypeRawValue,
                     color: request.color.messageColorRawValue,
-                    textAlignment: request.textAlignment.textAlignmentRawValue
+                    textAlign: request.textAlignment.textAlignmentRawValue
                 )
             )
+            
+            PayPalMessageConfig.setGlobalAnalytics(
+                integrationName: "BT_SDK",
+                integrationVersion: self.apiClient.metadata.version
```

Comment:
```suggestion
                integrationVersion: apiClient.metadata.version
```

Reply:
We need `self` because we're inside a closure (`fetchOrReturnRemoteConfiguration`)

Reply:
Update: I just reverted this call `PayPalMessageConfig.setGlobalAnalytics` due to an issue in PayPalMessaging, If we configure `integrationVersion` SDK returns an error

> issue: "INVALID_PARAMETER_SYNTAX"  
> description: "The value of a field does not conform to the expected format."

Once the MessagingTeam resolves this issue in the next version, we will be able to configure Message.GlobalAnalytics